src/utils.ts

```diff
@@ -8,3 +16,10 @@ export function isAccountPrivileged(sender: string): boolean {
     sender.endsWith(':matrix.parity.io') || sender.endsWith(':web3.foundation')
   );
 }
+
+export function convertAmountToBn(amount: string): bigint {
+  const parsedAmount = Number(amount);
+  return parsedAmount < MAX_ALLOWED_DRIP_FLOAT
+    ? BigInt(parsedAmount * 10 ** decimals)
```

Comment:
I think it would be better to convert the amount to BigInt first and then do the multiplications after. e.g. `parsedAmount * 10 ** decimals` might overflow [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) for some reason.

i.e.

```typescript
  const parsedAmount = BigInt(amount);
  // ...
```

Reply:
I was thinking about it, and played a lot with different ways before had to add bignum dependency. See my comment with screenshot of how it ends up to work with JS :(